.github/ISSUE_TEMPLATE/repo-create-application.md

```diff
@@ -0,0 +1,17 @@
+name: Repository creation
+description: Create a repository into KubeEdge Org
+title: "REQUEST: <Create> <github repo>"
+
+### 1. **Name for new repository:**
+
+
+ ### 2. **Who should be listed as approvers and reviewers in OWNERS?**
+
+
+### 3. **What should the repo description be?**
+
+
+### 4. **What SIG does this fall under?**
```

Comment:
nit: can we just use `labels: ["xxx"]` on the header instead? related to https://github.com/kubeedge/community/issues/159. this does not block this PR to be merged.